src/traces/parcoords/parcoords.js

```diff
@@ -218,13 +233,36 @@ function viewModel(model) {
             xScale: xScale,
             x: xScale(i),
             canvasX: xScale(i) * canvasPixelRatio,
-            unitScale: unitScale(height, c.verticalPadding),
+            // fixme remove the old unitScale
```

Comment:
`unitScale` is still being used for its inverse mapping due to how `y` values are arriving from the underlying `d3` interaction callback calls, but could in theory be unified, simplifying the current two-step map eg. `var y = d.unitScaleInOrder(d.unitScale.invert(d3.mouse(this)[1]));`